atlantic/sandy/Makefile

```diff
@@ -15,15 +15,16 @@ CLAWMAKE = $(CLAW)/clawutil/src/Makefile.common
 CLAW_PKG = geoclaw                  # Clawpack package to use
 EXE = xgeoclaw                 # Executable to create
 SETRUN_FILE = setrun.py        # File containing function to make data
-OUTDIR = _output               # Directory for output
+OUTDIR = _output              # Directory for output
 SETPLOT_FILE = setplot.py      # File containing function to set plots
-PLOTDIR = _plots               # Directory for plots
+PLOTDIR = _plots              # Directory for plots
 
 # Environment variable FC should be set to fortran compiler, e.g. gfortran
-FFLAGS += -DNETCDF -lnetcdf -I$(NETCDF4_DIR)/include
+FFLAGS += -DNETCDF -lnetcdff -I$(NETCDF4_DIR)/include -qopenmp -L$(NETCDF4_DIR)$ 
+
 
 # Need to add LFLAGS here explicitly otherwise will not be linked
-# LFLAGS += -lnetcdff -fopenmp
+LFLAGS += -lnetcdff -qopenmp -L$(NETCDF4_DIR)/lib
```

Comment:
This is specific to an intel compiler, probably want to make this generic by removing the extra flags except for the NETCDF part.